packages/cadl-rlc-emitter/src/modelUtils.ts

```diff
@@ -688,13 +689,13 @@ function mapCadlIntrinsicModelToTypeScript(
       return { type: "string", format: "time" };
     case "duration":
       return { type: "string", format: "duration" };
-    case "Map":
-      // We assert on valType because Map types always have a type
-      const valType = (cadlType as ModelType)?.properties.get("v");
-      return {
-        type: "object",
-        additionalProperties: getSchemaForType(program, valType!.type, usage)
-      };
+    // case "Map":
```

Comment:
I think Map changed Record

Reply:
Okay, let me double check.

Reply:
@joheredi I can't find record in the IntrinsicModelName definition here https://github.com/microsoft/cadl/blob/main/packages/compiler/core/types.ts#L178-L200.